catroid/src/main/java/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java

```diff
@@ -478,6 +480,16 @@
 			looksBrickList.add(new FlashBrick());
 		}
 
+		if (BuildConfig.FEATURE_WEBREQUEST_BRICK_ENABLED) {
+			if (!isBackgroundSprite) {
+				looksBrickList.add(new LookRequestBrick(context.getString(R.string.brick_look_request_default_value)));
+			} else if (ProjectManager.getInstance().getCurrentProject().getXmlHeader().islandscapeMode()) {
+				looksBrickList.add(new BackgroundRequestBrick(context.getString(R.string.brick_background_request_default_value_landscape)));
+			} else {
+				looksBrickList.add(new BackgroundRequestBrick(context.getString(R.string.brick_background_request_default_value_portrait)));
+			}
```

Comment:
The `BackgroundRequestBrick` will only be shown for Background Sprites. So its not possible to set the background from another Script. Is this specified like this? Because I tried and it would work to set the background from other scripts.

Reply:
Yes, that's by design. The brick is only supposed to be visible for background sprites.